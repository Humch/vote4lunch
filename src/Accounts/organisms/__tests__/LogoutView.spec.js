/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { BrowserRouter as Router } from 'react-router-dom';
import { LogoutView } from '../..';

describe('LogoutView Snapshot', () => {
  const initialState = { };
  const mockStore = configureStore();
  let store;

  beforeEach(() => {
    store = mockStore(initialState);
  });

  it('capturing Snapshot of LogoutView', () => {
    const renderedValue = renderer.create(<Router><LogoutView store={store} /></Router>).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
