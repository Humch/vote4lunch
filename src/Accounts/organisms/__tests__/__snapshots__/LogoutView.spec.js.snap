// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutView Snapshot capturing Snapshot of LogoutView 1`] = `null`;
